MediaBrowser.Controller/Library/IMediaSourceManager.cs

```diff
@@ -38,6 +38,27 @@ public interface IMediaSourceManager
         /// <returns>IEnumerable&lt;MediaStream&gt;.</returns>
         List<MediaStream> GetMediaStreams(MediaStreamQuery query);
 
+        /// <summary>
+        /// Gets the media attachments.
+        /// </summary>
+        /// <param name="">The item identifier.</param>
+        /// <returns>IEnumerable&lt;MediaAttachment&gt;.</returns>
```

Comment:
1. This doesn't return an `IEnumerable`
2. This should be an explanation of what is being returned, not the type.